xtask/src/opt.rs

```diff
@@ -202,3 +203,9 @@ pub struct FmtOpt {
     #[clap(long, action)]
     pub check: bool,
 }
+
+/// Run the auto-release process.
```

Comment:
Please add something like _This is tied to the GitHub CI._ or similar. We should also ensure that no unexpected things happen, when people run this locally.

Reply:
Updated to:
```
/// Run the auto-release process (should only be run by Github Action).
///
/// This is run by the `release.yml` workflow. It is not intended to be run
/// locally, and will exit immediately if `GITHUB_SHA` is not set.
```

So unless someone goes out of their way to set `GITHUB_SHA` it won't do anything when run locally, and even then it won't do publish anything unless that user has permissions to push to the repo/crates.io.